packages/loaders/stories/6-Spinner.stories.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Story, Meta } from '@storybook/react';
+import { Spinner, ISpinnerProps } from '@zendeskgarden/react-loaders';
+import { PALETTE } from '@zendeskgarden/react-theming';
+
+export default {
+  title: 'Components/Loaders/Spinner',
+  component: Spinner
+} as Meta;
+
+const StyledExampleWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+interface IDefaultStoryProps extends Omit<ISpinnerProps, 'size'> {
+  size: number;
+}
+
+export const Default: Story<IDefaultStoryProps> = ({ color, size, delayMS, duration }) => {
+  return (
+    <StyledExampleWrapper>
+      <Spinner color={color} size={`${size}px`} delayMS={delayMS} duration={duration} />
+    </StyledExampleWrapper>
+  );
+};
+
+Default.argTypes = {
+  size: {
+    name: 'size (px)',
+    control: { type: 'range', min: 30, max: 250, step: 1 }
+  },
+  duration: {
+    name: 'duration (ms)',
+    control: { type: 'range', min: 625, max: 2500, step: 10 }
+  },
+  color: { control: 'color' }
+};
+
+Default.args = {
+  size: 48,
+  duration: 1250,
+  color: PALETTE.blue[500]
```

Comment:
Ditto re: `size` and `color` starting out as `undefined`.

Reply:
With this not being a product environment, I'm not sure defaulting to these values shows the preferred default design. Match the defaults that we originally had in the knobs-examples feels like a more accurate solution to me (unless we apply the `font-size` and `color` to the example wrapper).

Reply:
Updated to align with the `Dots` example.